### PR TITLE
gl-1344_add_CollectInsertSizeMetrics_to_GDC_pipeline

### PR DESCRIPTION
**Pull request by nikellepetrillo**

### CODE REVIEW

**Review by khajoue2 — APPROVED** [review]

File: beta-pipelines/broad/somatic/single_sample/wgs/gdc_genome/GDCWholeGenomeSomaticSingleSample.wdl [hunk 1]
```diff
@@ -657,9 +685,19 @@ workflow GDCWholeGenomeSomaticSingleSample {
             bqsr_recal_file = gatk_baserecalibrator.bqsr_recal_file
     }
 
+    String output_bam_prefix = basename(gatk_applybqsr.bam, ".bam")
+
+    call collect_insert_size_metrics {
+        input:
+            input_bam = gatk_applybqsr.bam,
+            output_bam_prefix = "~{output_bam_prefix}.insert_size_histogram.pdf"
```
Comment (gbggrant) [inline beta-pipelines/broad/somatic/single_sample/wgs/gdc_genome/GDCWholeGenomeSomaticSingleSample.wdl:693]: Should the output_bam_prefix include ".insert_size_histogram.pdf"?

**Review by gbggrant — APPROVED** [review]
Looks good - one comment (likely typo).

**Review by jessicaway — APPROVED** [review]

File: beta-pipelines/broad/somatic/single_sample/wgs/gdc_genome/GDCWholeGenomeSomaticSingleSample.wdl (hunk 1 above)
Comment (jessicaway) [reply beta-pipelines/broad/somatic/single_sample/wgs/gdc_genome/GDCWholeGenomeSomaticSingleSample.wdl:693]: Good question. Will th resulting file end up with this included twice? ".insert_size_histogram.pdf.insert_size_histogram.pdf"

Comment (nikellepetrillo) [reply beta-pipelines/broad/somatic/single_sample/wgs/gdc_genome/GDCWholeGenomeSomaticSingleSample.wdl:693]: yup! thanks for catching @gbggrant. It doubled the file ext ".insert_size_histogram.pdf.insert_size_histogram.pdf"

**Pull request merged**